docs/modules/ROOT/pages/configuration.adoc

```diff
@@ -1000,6 +1000,16 @@ Sorbet:
   DocumentationExtension: .md
 ----
 
+== Setting the version tracking metadata for cops
+
+Each cop should gain the following additional metadata:
```

Comment:
```suggestion
Each cop can have the following additional metadata:
```

It might be useful to note that `VersionAdded` is required but the others are contingent.